homu/action.py

```diff
@@ -0,0 +1,210 @@
+import random
+from enum import Enum
+
+
+class LabelEvent(Enum):
+    APPROVED = 'approved'
+    REJECTED = 'rejected'
+    CONFLICT = 'conflict'
+    SUCCEED = 'succeed'
+    FAILED = 'failed'
+    TRY = 'try'
+    TRY_SUCCEED = 'try_succeed'
+    TRY_FAILED = 'try_failed'
+    EXEMPTED = 'exempted'
+    TIMED_OUT = 'timed_out'
+    INTERRUPTED = 'interrupted'
+    PUSHED = 'pushed'
+
+
+PORTAL_TURRET_DIALOG = ["Target acquired", "Activated", "There you are"]
+PORTAL_TURRET_IMAGE = "https://cloud.githubusercontent.com/assets/1617736/22222924/c07b2a1c-e16d-11e6-91b3-ac659550585c.png" # noqa
+
+
+class Action:
```

Comment:
I don't really understand the purpose of moving these functions to a class when there are no uses of `self` anywhere.

Reply:
I think moving all the actions to its own class makes the code more organized. Moreover, in the future, I'd also like to move all the logic that determines which action to execute outside of `parse_commands`, and have something like `action.execute(word, Auth.TRY, ...)`.

Reply:
I think I can see the value in your ideas for the future, but I don't think it makes sense to introduce the class in _this PR_ without making additional changes to make use of it. Does that make sense?

Reply:
I understand your concerns, but even without making use of this ideas yet, we're making `main.py` simpler moving out code that doesn't belong to that file. I don't see the drawbacks of moving that code out of `main.py`.

Reply:
I'm fine moving the functions into action.py. I don't think it needs to be in a class yet.

Reply:
Having all the methods in a class makes both the test and `main.py` much cleaner, as we don't need to import all the functions from `action.py`, just `from action import Action`, and we can [easily call tha actions](https://github.com/alexrs/homu/blob/991bbf78c099ccf7bc8562d3f9369f3155739331/tests/test_action.py#L19).

Reply:
I agree that it is easier to import a single class than all of the functions. I disagree that `Action().set_treeclosed(...)` is easier than `set_treeclosed(...)` :) The reason I am pushing back against the class right now is that I found it confusing when reading the change, and I would prefer not to introduce unnecessary confusion into the code when the only benefit is that it reduces the number of functions that need to be imported.